feat: DIA-2214: Add new Pdf object tag

hlomzik opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/15020818386)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- https://github.com/HumanSignal/label-studio-sdk/pull/452
<!-- Follow Merge -->


Can be used to display Pdf, only for classification.
```xml
<View>
  <Header value="Rate this article" />
  <Rating name="rating" toName="pdf" maxRating="10" icon="star" size="medium" />
  <Pdf name="pdf" value="$pdf" />
</View>
```

![Uploading Screenshot 2025-05-02 at 12.28.30.png…]()

hlomzik committed:
feat: DIA-2214: Add new Pdf object tag

Can be used to display Pdf, only for classification.

niklub commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/14837806076)
> Successfully merged:  create mode 100644 web/tools/extract-antd-no-reset/package.json

niklub committed:
Merge branch 'develop' into 'fb-dia-2214'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14837806076

Commit:
fmt

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14861521209

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-dia-2214'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14861521209

niklub commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/14882923105)
> Successfully merged:  create mode 100644 web/libs/core/src/lib/utils/query-client.ts

niklub committed:
Merge branch 'develop' into 'fb-dia-2214'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14882923105

Commit:
fix generator

niklub commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/14942719260)
> Successfully merged:  42 files changed, 590 insertions(+), 180 deletions(-)

niklub committed:
Merge branch 'develop' into 'fb-dia-2214'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14942719260

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14973390606

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-dia-2214'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14973390606

niklub commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/15017758857)
> Successfully merged:  13 files changed, 212 insertions(+), 228 deletions(-)

niklub committed:
Merge branch 'develop' into 'fb-dia-2214'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/15017758857

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/15020818386
